GetJobber/atlantis
# feat(components-native): Migrate  Input file for android

## Motivations

Split out iOS to reduce size and scope of PR

## Changes



### Added

- Added InputFile to components native for Android

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

- Install pre-release
- Verify InputFile on Android
- Test PR [here](https://github.com/GetJobber/jobber-mobile/pull/7509)

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

![Random photo of Atlantis](https://loremflickr.com/672/400/atlantis)
